Source/Scene/ImageryLayerCollection.js

```diff
@@ -325,6 +325,105 @@ ImageryLayerCollection.prototype.lowerToBottom = function (layer) {
 
 var applicableRectangleScratch = new Rectangle();
 
+/**
+ * Determines the imagery layers that are intersected by a pick ray. To compute a pick ray from a
+ * location on the screen, use {@link Camera.getPickRay}.
+ *
+ * @param {Ray} ray The ray to test for intersection.
+ * @param {Scene} scene The scene.
+ * @return {ImageryLayerCollection|undefined} An `ImageryLayerCollection` that includes all of
```

Comment:
I like the thought of returning an `ImageryLayerCollection` but I think an array of `ImageryLayer`'s is fine. This matches `drillPick` and `pickImageryLayerFeatures` more closely.